build/MUX-PGOInstrument.yml

```diff
@@ -56,10 +39,40 @@ jobs:
     helixType: 'test/frpkg'
     dependsOn: CreateNugetPackage
     pkgArtifactPath: '$(artifactDownloadPath)\drop\FrameworkPackage'
+    matrix:
+      Release_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Release'
+      Release_x64:
+        buildPlatform: 'x64'
+        buildConfiguration: 'Release'
+    test_matrix:
+      Release_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Release'
+        helixTargetQueues: 'windows.10.amd64.client19h1.open.xaml'
+      Release_x64:
+        buildPlatform: 'x64'
+        buildConfiguration: 'Release'
+        helixTargetQueues: 'windows.10.amd64.client19h1.open.xaml'
 
 - template: AzurePipelinesTemplates\MUX-ProcessTestResults-Job.yml
   parameters:
-    dependsOn:
-    - RunNugetPkgTestsInHelix
-    - RunFrameworkPkgTestsInHelix
-    rerunPassesRequiredToAvoidFailure: 5
\ No newline at end of file
+    dependsOn: RunFrameworkPkgTestsInHelix
+    rerunPassesRequiredToAvoidFailure: 5
+    minimumExpectedTestsExecutedCount: 10
+
+- template: AzurePipelinesTemplates\MUX-MergePGD-Job.yml
+  parameters:
+    dependsOn: ProcessTestResults
+    matrix:
```

Comment:
Do these have to happen in parallels? There is a small overhead in creating a job (finding an available agent, cloning the repo, etc.), so it might be better if a single job could run merge for both x86 and x64. Further this job could then be merged with the next 'publish' job. I don't think there is a strong reason to split this work across multiple jobs.